docs/install.rst

```diff
@@ -65,12 +71,26 @@ download and run it as follows::
   user@webserver:~$ ./certbot-auto --help
 
 .. hint:: The certbot-auto download is protected by HTTPS, which is pretty good, but if you'd like to
-          double check the integrity of the ``certbot-auto`` script, you can use these steps for verification before running it::
+          double check the integrity of the ``certbot-auto`` script, you can use these steps for verification before running it using gpg2::
```

Comment:
I think that reiterating the use of `gpg2`here is unnecessary, as it will be clear from the example below, and it was described for the user in the previous paragraph that was added.